tests/jquery.flot.navigate.Test.js

```diff
@@ -509,7 +397,206 @@ describe("flot navigate plugin", function () {
             expect(yaxis.min).toBe(-10);
             expect(yaxis.max).toBe(0);
         });
+    });
 
+    describe('touchZoom', function() {
+
+        function getDistance(coords) {
+                return Math.sqrt((coords[0].x - coords[1].x) * (coords[0].x - coords[1].x) + ((coords[0].y - coords[1].y) * (coords[0].y - coords[1].y)));
+            }
```

Comment:
Move this helper function at the bottom of the section. Also check the indentation.